index.html

```diff
@@ -2433,21 +2455,55 @@ <h2>
           <ol class="algorithm">
             <li>Set the [=assigned launch consumer=] to |consumer|.
             </li>
-            <li>If the [=unconsumed launch params=] on the {{Window/window}} is
-            set:
+            <li>[=list/For each=] [=unconsumed launch params=]:
+            </li>
+            <li>
+              <ol>
+          <li>Call |consumer| with [=unconsumed launch params=].
+          </li>
+              </ol>
+            </li>
+            <li>Clear the list of [=unconsumed launch params=].
+            </li>
+          </ol>
+        </section>
+      </section>
+      <section>
+        <h3>
+          Handling Web App Launches
+        </h3>
+        <p>
+          The steps to <dfn>launch a web app</dfn> are given by the 
+          following algorithm. The algorithm takes {{LaunchParams}}
+          |params:LaunchParams|.
+        </p>
+        <ol class="algorithm">
+          <li>If |params:LaunchParams| is null, set |params:LaunchParams| to
```

Comment:
Is it typical to repeatedly specify the type in (i.e. |params:LaunchParams| here instead of just |params|)? I hadn't been doing that in the algorithms I added for file handling.

Reply:
I see this pattern done in: https://www.w3.org/TR/appmanifest/#dfn-processing-a-manifest

Reply:
I've changed it to match what you've been doing Evan.